draft-twi-reference-architecture.md

```diff
@@ -49,22 +49,44 @@ informative:
 
-1. Lack of workload isolation against the hardware and the operating system owners/administrators, as well as peer workload instances
-2. Lack of strong binding between a workload credential and the workload instance to which that credential had been issued
-3. Inability to associate a credential with a set of decisions leading up to its issuance
+1. Lack of Workload isolation against the hardware and the operating system owners/administrators, as well as peer Workload instances
+2. Lack of strong binding between a Workload Credential and the Workload instance to which that Credential had been issued
+3. Inability to associate a Credential with a set of decisions leading up to its issuance
 _Note that these requirements are related: lack of process isolation eases credential exfiltration and leads to credential leakage and reuse._
 
-In the immediate term, effective Confidential Computing faces a significant challenge: the scale of uplift and immature tooling are both clear barriers. Longer term, however, Confidential Computing provides a vital improvement due to its unique features and broad availability. Confidential Computing-assisted mechanisms have to fit inside the emerging Workload Identity solution ecosystem. The evolution of this ecosystem should remain in alignment with the expectations of the owners and operators of Confidential Computing workloads. These efforts will build on the concept of _Trustworthy Workload Identity_ defined below. Data-in-use protection of workloads that have such identities will be a critical downstream effect.
+The Confidential Computing Consortium defines Confidential Computing as "protection of data in use by performing computation in a hardware-based, attested Trusted Execution Environment (TEE)". In the immediate term, effective Confidential Computing faces a significant challenge: the scale of uplift and immature tooling are both clear barriers. Longer term, however, Confidential Computing provides a vital improvement due to its unique features and broad availability. Confidential Computing-assisted mechanisms have to fit inside the emerging Workload Identity solution ecosystem. The evolution of this ecosystem should remain in alignment with the expectations of the owners and operators of Confidential Computing Workloads. These efforts will build on the concept of _Trustworthy Workload Identity_ defined further below. Data-in-use protection of Workloads that have such Identities will be a critical downstream effect.
+
+While it is quite clear that for the foreseeable future this need is not going to be met by Confidential Computing alone, the security features related to Workload identification and isolation offered by Confidential Computing eclipse other approaches. Confidential Computing-strengthened Workload Identity will need to be designed to fit within, and ideally serve as the North Star of Workload Identity as it becomes a foundational pillar of trustworthy and governable enterprise computing.
```

Comment:
use of "this". Repetition of the problem statement is fine.

Reply:
> use of "this". Repetition of the problem statement is fine.

What do you mean by 'use of "this"'?